docs/README.md

```diff
@@ -606,12 +611,17 @@ onOpen = successCallback
 
 <!-- tabs:end -->
 
-## on-close
+## on-close  ( *optional* )
```

Comment:
Let's remove the `( optional )` bit here as it's unclear _what_ is optional to somebody reading this for the first time.